src/ui/MobileMenu/MobileEmojisBottomSheet.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { ReactElement, useMemo, useState } from 'react';
+import { EmojiContainer } from '@sendbird/chat';
+import { Member } from '@sendbird/chat/groupChannel';
+import { FileMessage, Reaction, UserMessage } from '@sendbird/chat/message';
+
+import './mobile-menu-emojis.scss';
+
+import BottomSheet from '../BottomSheet';
+import { getEmojiUrl } from '../../utils';
+import ImageRenderer from '../ImageRenderer';
+import Icon, { IconColors, IconTypes } from '../Icon';
+import Label, { LabelColors, LabelTypography } from '../Label';
+import { useChannelContext } from '../../modules/Channel/context/ChannelProvider';
```

Comment:
`MobileEmojisBottomSheet` wont be used inn thread?

Reply:
So I changed it to get the channel through props.
btw I will make the hybrid hook function to get context from multiple context provider